secant/Screens/Loading/LoadingScreen.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  LoadingScreen.swift
+//  secant-testnet
+//
+//  Created by Francisco Gindre on 9/2/21.
+//
+
+import SwiftUI
+
+protocol LoadingScreenRouter: AnyObject {
+    func proceedToHome()
+    func failWithError()
+    func proceedToWelcome()
+}
+
+struct LoadingScreen: View {
+    @State var router: LoadingScreenRouter?
+    
+    @StateObject var viewModel: LoadingScreenViewModel
+    
+    var body: some View {
+        Text("Loading")
+            .onReceive(viewModel.$loadingResult, perform: { result in
```

Comment:
What do you think of this formatting to keep things cleanly indented and easy to read?  This is in line with what I've fixed in the Swfitlint PR.

```
.onReceive(
    viewModel.$loadingResult,
    perform: { result in
        guard
            let loadingResult = result,
            let router = self.router
        else { return }

        viewModel.callRouter(router, with: loadingResult)
     }
)
.onAppear {
    DispatchQueue.main.asyncAfter(deadline: .now() + 4) {
        viewModel.loadAsync()
    }
}
```

Reply:
Yes this is much better